pkg/administration/events/event.go

```diff
@@ -3,7 +3,9 @@ package events
 import (
 	"strings"
 
-	gogoTimestamp "github.com/gogo/protobuf/types"
+	// TODO: find a way to break the dependency loop when pulling pkg/protoconv
```

Comment:
yeah I don't think the logging library should depend on anything outside of itself and the standard library, but that's unrelated to this PR

Reply:
How about removing this logger from converting function?

https://github.com/stackrox/stackrox/blob/6ed966ddbe6c1c22876fdfac6de6c555445ae0ff/pkg/protoconv/time.go#L57-L61